scripts/valhalla_build_elevation

```diff
@@ -208,24 +246,45 @@ def download(tile: Tile, output_dir, decompress):
     LOGGER.info(f"Downloading tile {tile.name}")
     try:
         with request.urlopen(url) as res, open(filepath, "wb") as f:
-            if decompress:
-                with gzip.GzipFile(fileobj=res, mode="rb") as gz:
-                    f.write(gz.read())
-            else:
+            if compression is TileCompression.GZIP:
                 f.write(res.read())
+            else:
+                with gzip.GzipFile(fileobj=res, mode="rb") as gz:
+                    uncompressed = gz.read()
+                    if compression is TileCompression.UNCOMPRESSED:
+                        f.write(uncompressed)
+                    elif compression is TileCompression.LZ4:
+                        # Local imports are yuck, even though it only takes effect once. the alternative is
+                        # requiring everyone to have the LZ4 package available, regardless of whether the user needs it.
+                        # Compression level 6 was chosen after some benchmarking as having a reasonable balance between
+                        # compression time and space savings. The end result will be larger than the maximally gzipped
+                        # tiles from AWS, but only by around 12%. To narrow the gap to ~10% requires compression level
+                        # 9, and more than double the compression time, so this is the approximate efficient frontier.
+                        # for our purposes. Note that in general LZ4 decompression cost does not vary much with
+                        # compression level, unless using extremely high compression levels.
```

Comment:
local import is perfect here IMO, we do the same already with shapely in case of geojson downloads. I'd cut the comment a little maybe :sweat_smile: